### ISSUE TITLE
target page exclamation points have no border in Edge high contrast mode

### ISSUE THREAD

**Issue by karanbirsingh** [microsoft/accessibility-insights-web#3272]
## Describe the bug

When in Windows high contrast mode in Edge, the exclamation points on the target page can be hard to make out because there's no border separating the black background of the icon from the page. 
![high contrast exclamation point](https://user-images.githubusercontent.com/7775527/90930186-f73ce800-e3ae-11ea-81cc-f782a39e5621.png)

**To Reproduce**

- Run the extension in Edge
- Run fastpass on any page

## Context (please complete the following information)
Windows 1909 Build 18363.1016
Edge Version 84.0.522.63 (Official build) (64-bit)
Extension 2.20.0

**Comment by ferBonnin** [comment]
We should add a border to the box containing the target page exclamation point, if possible, doing inverse colors  (white box and black exclamation mark) would be a better alternative.

**Comment by lisli1** [comment]
Also noting here that the button icons aren't visible in the issues dialog

Screenshot of issues dialog buttons with non-visible icons
![image](https://user-images.githubusercontent.com/48259897/115631213-3adb4f00-a2ba-11eb-8c64-10641301c210.png)

**Comment by DaveTryon** [comment]
@karanbirsingh, this is now in Canary and Insider rings

**Comment by karanbirsingh** [comment]
looks great, thanks @DaveTryon !
